### PR TITLE
Issues fix from github issues ( homepage, wrangler, header) + favicon

### LINKED ISSUE #9: Navigation menu drop down should have a hover state.

### LINKED ISSUE #14: Merge View all Connectors and View all transformations buttons into a single View all Plugins button
image

Rename the view all connectors button to view all plugins. Remove the view all transformations button.

### LINKED ISSUE #15: CDAP.io - home page title capitalization is inconsistent
The capitalization of all the titles in the home page is inconsistent.
Some titles use setence capitalization
e.g. This is a title
Other titles use title capitalization
e.g. This Is A Title

### LINKED ISSUE #17: CDAP.io - Home page Accellerator cards text is truncated
The text in the accellerator card is truncated

e.g. 
**Preparation**
Preparation allows you to visually and interactively cleanse and prepare raw data, with the aim of making it consumable for further processing. It provides a standardized UI driven interactive flow that takes the pain out of

Please edit text appropriately

### LINKED ISSUE #19: CDAP.io - Get Started - capitalization of titles is inconsistent
Please review and make the capiralization of the titles consistent.

### LINKED ISSUE #24: Preparation - Change to "Wrangler"
We need to change product name from "Preparation" to "Wrangler". This is a global change

### PR DESCRIPTION
**Pull request by dbuzhorkaa**
* closes #24, closes #19, closes #17, closes #14, closes #15, closes #14, closes #9

### CODE REVIEW

File: data/en/homepage.json [hunk 1]
```diff
@@ -70,34 +70,34 @@
...
         },
         {
-          "header": "Extensible and reusable",
+          "header": "Extensible and Reusable",
           "paragraph": "Templates and blueprints for common use-cases; Hub for sharing pre-built plugins, applications and solutions; Extensible APIs for security, metadata, runtimes and storage.",
           "id": "feature_3"
         },
         {
-          "header": "Hybrid and multi cloud",
+          "header": "Hybrid and Multi Cloud",
```
Comment (bdmogal) [inline data/en/homepage.json]: Hybrid and multi-cloud

File: data/en/homepage.json [hunk 2]
```diff
@@ -70,34 +70,34 @@
...
       "right_column": [
         {
-          "header": "Portable runtime environments",
+          "header": "Portable Runtime Environments",
           "paragraph": "Build once, run anywhere through portability across runtime environments such as Kubernetes, Apache Hadoop YARN and Docker.",
           "id": "feature_5"
         },
         {
-          "header": "Extensible and reusable",
+          "header": "Extensible and Reusable",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert this change

File: data/en/homepage.json [hunk 3]
```diff
@@ -70,34 +70,34 @@
...
-          "header": "Seamless operations",
+          "header": "Seamless Operations",
           "paragraph": "REST APIs and CLI for every interaction; Time and process based scheduling; Standardized logs and metrics for all execution environments.",
           "id": "feature_2"
         }
       ],
       "right_column": [
         {
-          "header": "Portable runtime environments",
+          "header": "Portable Runtime Environments",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert this

File: data/en/homepage.json [hunk 4]
```diff
@@ -70,34 +70,34 @@
...
         },
         {
-          "header": "Enterprise ready",
+          "header": "Enterprise Ready",
           "paragraph": "Metadata repository with automatic technical and operational metadata capture; Business metadata annotations; Data discovery through search based on metadata; Data governance with dataset and field level lineage and auditing; Integration with enterprise security systems.",
           "id": "feature_4"
         },
         {
-          "header": "Seamless operations",
+          "header": "Seamless Operations",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert this

File: data/en/homepage.json [hunk 5]
```diff
@@ -70,34 +70,34 @@
...
       "left_column": [
         {
-          "header": "Rapid development",
+          "header": "Rapid Development",
           "paragraph": "Developer SDK and APIs with abstractions over common data processing patterns; Sandbox mode, programmatic and UI driven debugging; In-memory mode and testing framework to simplify testing; Support for cutting edge Cloud, Apache Hadoop and Apache Spark technologies.",
           "id": "feature_1"
         },
         {
-          "header": "Enterprise ready",
+          "header": "Enterprise Ready",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert this

File: data/en/homepage.json [hunk 6]
```diff
@@ -70,34 +70,34 @@
       "title": "CDAP features",
       "left_column": [
         {
-          "header": "Rapid development",
+          "header": "Rapid Development",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert

File: data/en/homepage.json [hunk 7]
```diff
@@ -39,25 +39,25 @@
...
         {
-          "header": "Increased flexibility",
+          "header": "Increased Flexibility",
           "icon": "images/icons/icon-3.svg",
           "paragraph": "CDAP is 100% open source, portable and extensible. It integrates with latest Big Data and Cloud technologies.",
           "id": "increased-flexibility"
         },
         {
-          "header": "Improved visibility",
+          "header": "Improved Visibility",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert

File: data/en/homepage.json [hunk 8]
```diff
@@ -12,7 +12,7 @@
     {
       "id": "profile",
       "tabTitle": "On Premises",
-      "title": "Get Started on premises",
+      "title": "Get Started on Premises",
```
Comment (bdmogal) [inline data/en/homepage.json]: Get started on-prem

File: data/en/homepage.json [hunk 9]
```diff
@@ -39,25 +39,25 @@
       "description": "CDAP lets developers, business analysts and data scientists focus on insights, analytics and business value instead of wrestling with infrastructure, and integration.",
       "items": [
         {
-          "header": "Reduced complexity",
+          "header": "Reduced Complexity",
```
Comment (bdmogal) [inline data/en/homepage.json]: revert

File: data/en/homepage.json [hunk 10]
```diff
@@ -39,25 +39,25 @@
...
         {
-          "header": "Increased velocity",
+          "header": "Increased Velocity",
           "icon": "images/icons/icon-2.svg",
           "paragraph": "CDAP platform and intuitive UI allow, rapid onboarding and faster time to production. It provides business value, by reducing repetition and silos, and promoting reuse and self service: build once, run anywhere",
           "id": "increased-velocity"
         },
         {
-          "header": "Increased flexibility",
+          "header": "Increased Flexibility",
```
Comment (bdmogal) [inline data/en/homepage.json]: ditto

File: data/en/homepage.json [hunk 11]
```diff
@@ -39,25 +39,25 @@
...
         {
-          "header": "Reduced complexity",
+          "header": "Reduced Complexity",
           "icon": "images/icons/icon-1.svg",
           "paragraph": "CDAP lowers barrier to entry to complex technologies. It increases developer productivity through focus on business logic; and leads to accelerated ROI",
           "id": "reduced-complexity"
         },
         {
-          "header": "Increased velocity",
+          "header": "Increased Velocity",
```
Comment (bdmogal) [inline data/en/homepage.json]: ditto

File: data/en/homepage.json [hunk 12]
```diff
@@ -159,7 +159,9 @@
     }
   ],
   "companies": {
-    "title": "Companies using CDAP",
+    "title": "Companies Using CDAP",
```
Comment (bdmogal) [inline data/en/homepage.json]: ditto

**Comment by bdmogal** [comment]
#24 #9 and #14 are good to go.

Regarding capitalization of titles:
1. We should use sentence case everywhere. See comments.
2. For buttons, its ok to use title case 

@leacd please let me know if you agree.

File: data/en/homepage.json (hunk 1 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 2 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 3 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 4 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 5 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 6 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 7 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 8 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 9 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 10 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 11 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

File: data/en/homepage.json (hunk 12 above)
Comment (dbuzhorkaa) [reply data/en/homepage.json]: done

**Comment by dbuzhorkaa** [comment]
Addressed all comments. @bdmogal please recheck
